addons/project/views/project_project_views.xml

```diff
@@ -153,11 +153,13 @@
                                     </div>
                                 </group>
                             </group>
-                        </page>
-                        <page name="analytic" string="Analytic" groups="analytic.group_analytic_accounting">
                             <group>
```

Comment:
This group will be added empty  when only the project module is installed and the analytic feature is not active.